sistema/main.php

```diff
@@ -53,6 +53,12 @@
             <li><a href="#">Running</a></li>
             <li><a href="#">Tenis</a></li>
         </ul>
+
+        <header>Administrador</header>
+        <hr>
+        <ul>
+            <li><a href="users.php">Usuarios</a></li>
+        </ul>
     </div>
 
```

Comment:
Fields in here were deleted, because of the changes implemented by Axel on the main branch